app/views/idv/shared/_failure_to_proof_url.html.slim

```diff
@@ -0,0 +1,9 @@
+- if decorated_session.sp_name
+  hr
+    .mb2.mt2
+      .right = link_to image_tag(asset_url('carat-right.svg'), size: '10'),
+        decorated_session.failure_to_proof_url, class: 'bold block btn-link text-decoration-none'
+      = link_to raw('Get help at <strong>' + decorated_session.sp_name + '</strong>'),
```

Comment:
Also, `Get help at` should be moved into a translation. That should mean you are able to do this:

```ruby
t('path.to.translation_html', sp_name: decorated_session.sp_name)`
```

And then you translation would have the `strong` tags in it and they shouldn't be escaped since the translation is prefixed with `html`.

Reply:
Here is an example of this in our code:

The translation: https://github.com/18F/identity-idp/blob/master/config/locales/devise/en.yml#L123

Where it is used in the presenter: https://github.com/18F/identity-idp/blob/master/app/presenters/two_factor_auth_code/max_attempts_reached_presenter.rb#L47-L51